<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_logowanie_do_banku">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>361</width>
        <height>49</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Logowanie do Banku</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bankomat">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>221</width>
        <height>49</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bankomat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>761</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Login: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Hasło:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_login">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2_password">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_login">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_zarejestruj_sie">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>220</y>
         <width>261</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zarejestruj się</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_anuluj_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>480</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>611</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_miasto"/>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_ulica"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Imie:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_nazwisko"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ulica i numer domu:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Miasto:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_login_2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Numer Telefonu:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_imie"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Login:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Adres email:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_numer_telefonu"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_adres_email"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Nazwisko:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hasło:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_haslo_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rejestracja</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_zarejestruj_sie_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>500</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Zarejestruj się</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_anuluj">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>500</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="pushButton_anuluj_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Numer konta(id):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Kwota:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_numer_konta_id_bankomat"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_kwota_bankomat"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_wplata">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Wpłata</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_wyplata">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Wypłata</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBank_muj">
    <property name="title">
     <string>muj Bank</string>
    </property>
   </widget>
   <addaction name="menuBank_muj"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
